### PR TITLE
[ES] Clean up `Twitter`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Twitter/es.md [hunk 1]
```diff
@@ -15,11 +15,11 @@ osu! tiene bastantes cuentas de Twitter, cada una para diferentes propósitos. L
...
+| ![NAT avatar](img/osu_nat.png) | [@osu_nat](https://twitter.com/osu_nat "Twitter") | Noticias, anuncios, y breves encuestas comunitarias realizadas por el [NAT](/wiki/People/Nomination_Assessment_Team) (no manejado por [el osu! team](/wiki/People/osu!_team)). |
+| ![pp committee avatar](img/ppcommittee.png) | [@pp_committee](https://twitter.com/pp_committee) | Anuncios de cálculo de dificultad para todos los modos de juego, manejado por el [comité de puntos de rendimiento](/wiki/People/Performance_Points_Committee). |
 
 ## Personal
 
 | Avatar | Usuario | Descripción |
 | :-: | :-: | :-- |
-| ![Dean Herbert avatar](img/ppy.jpg?2) | [@ppy](https://twitter.com/ppy "Twitter") | La cuenta personal del [creador de osu!](/wiki/People/peppy), que no es estrictamente sobre osu!, pero de todas maneras cubre la mayoría de ella. |
+| ![Dean Herbert avatar](img/ppy.jpg?2) | [@ppy](https://twitter.com/ppy "Twitter") | La cuenta personal de Twitter del [creador de osu!](/wiki/People/peppy), que no es estrictamente sobre osu!, pero de todas maneras cubre la mayoría de ella. |
```
Comment (MChecaH) [inline wiki/Twitter/es.md:25]: In the original article, the last `it` is making reference to the game, not the account. If we're implicitly going to refer to it (the game), it'd be more correct if the correct gendered pronoun was used (El Juego, not La Juego).
```suggestion
| ![Dean Herbert avatar](img/ppy.jpg?2) | [@ppy](https://twitter.com/ppy "Twitter") | La cuenta personal de Twitter del [creador de osu!](/wiki/People/peppy), que no es estrictamente sobre osu!, pero de todas maneras cubre la mayoría de él. |
```

File: wiki/Twitter/es.md [hunk 2]
```diff
@@ -6,20 +6,20 @@ osu! tiene bastantes cuentas de Twitter, cada una para diferentes propósitos. L
 
 | Avatar | Usuario | Descripción |
 | :-: | :-: | :-- |
-| ![osu!status avatar](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus "Twitter") | Notificaciones de problemas relacionados con el sitio web y [Bancho](/wiki/Bancho_(server)). |
-| ![osu! support avatar](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport "Twitter") | Ayuda con tu cuenta y/o problemas de la comunidad. Manejado por el [equipo de soporte de cuentas](/wiki/People/Account_support_team). |
+| ![Avatar de osu!status](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus "Twitter") | Notificaciones de problemas relacionados con el sitio web y [Bancho](/wiki/Bancho_(server)). |
+| ![Avatar de osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport "Twitter") | Ayuda con tu cuenta y/o problemas de la comunidad. Manejado por el [equipo de soporte de cuentas](/wiki/People/Account_support_team). |
```
Comment (UnknownOsu) [inline wiki/Twitter/es.md:10]:
```suggestion
| ![Avatar de osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport "Twitter") | Ayuda con tu cuenta y/o problemas de la comunidad. Manejada por el [equipo de soporte de cuentas](/wiki/People/Account_support_team). |
```

File: wiki/Twitter/es.md [hunk 3]
```diff
@@ -6,20 +6,20 @@ osu! tiene bastantes cuentas de Twitter, cada una para diferentes propósitos. L
...
-| ![NAT avatar](img/osu_nat.png) | [@osu_nat](https://twitter.com/osu_nat "Twitter") | Noticias, anuncios, y breves encuestas comunitarias realizadas por el [NAT](/wiki/People/Nomination_Assessment_Team) (no manejado por [el osu!team](/wiki/People/osu!_team)). |
-| ![pp committee avatar](img/ppcommittee.png) | [@pp_committee](https://twitter.com/pp_committee) | Anuncios de cálculo de dificultad para todos los modos de juego, manejado por el  [Comité de Puntos de Rendimiento](/wiki/People/Performance_Points_Committee). |
+| ![Avatar de osu!](img/osugame.jpg) | [@osugame](https://twitter.com/osugame "Twitter") | La fuente oficial de noticias y anuncios. |
+| ![Avatar de BanchoBoat](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat "Twitter") | Chistes cuando las cosas salen mal. |
+| ![Avatar del NAT](img/osu_nat.png) | [@osu_nat](https://twitter.com/osu_nat "Twitter") | Noticias, anuncios, y breves encuestas comunitarias realizadas por el [NAT](/wiki/People/Nomination_Assessment_Team) (no manejado por [el osu! team](/wiki/People/osu!_team)). |
```
Comment (UnknownOsu) [inline wiki/Twitter/es.md:18]:
```suggestion
| ![Avatar del NAT](img/osu_nat.png) | [@osu_nat](https://twitter.com/osu_nat "Twitter") | Noticias, anuncios, y breves encuestas comunitarias realizadas por el [NAT](/wiki/People/Nomination_Assessment_Team) (no manejada por [el osu! team](/wiki/People/osu!_team)). |
```

File: wiki/Twitter/es.md [hunk 4]
```diff
@@ -6,20 +6,20 @@ osu! tiene bastantes cuentas de Twitter, cada una para diferentes propósitos. L
...
+| ![Avatar del commité de pp](img/ppcommittee.png) | [@pp_committee](https://twitter.com/pp_committee) | Anuncios de cálculo de dificultad para todos los modos de juego, manejado por el [comité de puntos de rendimiento](/wiki/People/Performance_Points_Committee). |
```
Comment (UnknownOsu) [inline wiki/Twitter/es.md:19]:
```suggestion
| ![Avatar del commité de pp](img/ppcommittee.png) | [@pp_committee](https://twitter.com/pp_committee) | Anuncios de cálculo de dificultad para todos los modos de juego, manejada por el [comité de puntos de rendimiento](/wiki/People/Performance_Points_Committee). |
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

**Review by cl8n — APPROVED** [review]

**Pull request merged**